pkg/cmd/pr/shared/commentable.go

```diff
@@ -117,10 +119,32 @@ func CommentableRun(opts *CommentableOptions) error {
 		return err
 	}
 	apiClient := api.NewClientFromHTTP(httpClient)
-	params := api.CommentCreateInput{Body: opts.Body, SubjectId: commentable.Identifier()}
-	url, err := api.CommentCreate(apiClient, repo.RepoHost(), params)
-	if err != nil {
-		return err
+	url := ""
+	if opts.EditLast {
+		username, err := api.CurrentLoginName(apiClient, repo.RepoHost())
```

Comment:
`api.CurrentLoginName` is retrieved as "github-actions[bot]" when executed with GitHub Actions `GITHUB_TOKEN: ${{ github.token }}`, but [Comment.Author.Login](https://docs.github.com/en/graphql/reference/interfaces#actor) will be retrieved under a different name, "github-actions", so there is no match.
Should I trim "[bot]" if it is "github-actions[bot]"?

Reply:
When creating a PAT, `--edit-last` works fine, but when using `github.token` in CI, there is a problem with the username not matching and not getting the last comment.
I don't know why the names don't match, should I create an issue?

Reply:
@seachicken What is the error you are receiving? Is it showing the incorrect user or not retrieving any user at all?